source/reference/api/start.txt

```diff
@@ -406,4 +439,21 @@ To check whether it is safe to rename collections, call the
   Renaming the collection from this point on does not block
   its sharding on the destination cluster, even in the event  of a crash.
 
+.. _c2c-required-indexes:
+
+Required Indexes
+~~~~~~~~~~~~~~~~
+
+When you call ``/start`` with the ``buildIndexes`` option set to ``never``,
+``mongosync`` skips building unnecessary indexes.  
+
+Indexes that are always built include:
+
+* ``mongosync`` automatically builds an index for the ``_id`` field on every
```

Comment:
"An index on the ``_id`` field is automatically built for every collection mongosync copies"

Reply:
Made minor wordsmithing here, but left the clauses in their previous order since flipping them would shift the voice from active to passive.